stdlib/2/subprocess.pyi

```diff
@@ -57,10 +57,13 @@ STDOUT = ...  # type: int
 
 class CalledProcessError(Exception):
     returncode = 0
-    cmd = ...  # type: str
-    output = ...  # type: str  # May be None
+    cmd = ...  # type: Any
+    output = ...  # type: Any
 
-    def __init__(self, returncode: int, cmd: str, output: Optional[str] = ...) -> None: ...
+    def __init__(self,
+                 returncode: int,
+                 cmd: Union[bytes, Text, List[bytes], List[Text]],
```

Comment:
Sure, please do. Also, I think the argument should be `Sequence[Union[bytes, Text]]`, since mixing bytes and str seems to work fine. Using a Union as the type argument is OK because Sequence is covariant.